data/www/index-ie.html

```diff
@@ -0,0 +1,22 @@
+﻿<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="Title" content="E4d@box Marlin WEBUI" />
+    <meta name="Description" content="Marlin custom webUI no esplib no esp3d" />
+    <meta name="Generator" content="E4d@box" />
+    <meta name="Owner" Content="E4d@box" />
+    <meta name="Author" Content="MandrakeDesign" />
+    <meta name="google" content="notranslate" />
+    <meta http-equiv="Content-Language" content="en_GB" />
+    <title>E4d@box Marlin WEBUI</title>
+    <link rel="shortcut icon" href="#" />
+  </head>
+  <body>
+    <div style="padding-top:100px; text-align:center">
+      <h1>Sorry, but MS Internet Explorer browser is not supported by this web application.<br>Please upgrade to Edge or use another browser</h1>
```

Comment:
What prevents IE support? What percentage of Windows users uses IE as their default browser?